spec/fetch.spec.js

```diff
@@ -241,8 +241,8 @@ describe('test trimID method for npm and git', function () {
                 expect(fs.existsSync(result)).toBe(true);
                 expect(result).toMatch('cordova-plugin-device');
 
-                // refetch to trigger trimID
-                return fetch('https://github.com/apache/cordova-plugin-media.git', tmpDir, opts);
+                // refetch to trigger trimID, with shortcode URL
+                return fetch('github:apache/cordova-plugin-media', tmpDir, opts);
```

Comment:
testing of the old/other way can be removed?

Reply:
I think we probably want to support both, since some people are using `https://` URLs for git projects, or `file:///` URLs for local plugins.

Reply:
Yep, but didn't this just remove one of the tests?

Reply:
That fetch call is fine to update. Since it isn't testing the url but making sure trimID is handling refetching

Reply:
Thanks for the explanation @stevengill